store/pegasus.go

```diff
@@ -0,0 +1,299 @@
+package store
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/XiaoMi/pegasus-go-client/admin"
+	"github.com/XiaoMi/pegasus-go-client/pegasus"
+	"github.com/spf13/cast"
+	"strings"
+	"time"
+)
+
+const (
+	// PegasusType represents the storage type as a string value
+	PegasusType = "pegasus"
+	// PegasusTagPattern represents the tag pattern to be used as a key in specified storage
+	PegasusTagPattern = "go_cache_tag_%s"
```

Comment:
Could you please use « gocache_tag_%s » please? Same for DefaultTable below.

Reply:
ok, i will do it